pilco/policies/eq_policy.py

```diff
@@ -0,0 +1,256 @@
+from pilco.policies.policy import Policy
+
+import numpy as np
+
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+from not_tf_opt import UnconstrainedVariable
+
+tfd = tfp.distributions
+
+
+class EQPolicy(Policy):
+
+    def __init__(self,
+                 state_dim,
+                 action_dim,
+                 num_eq_features,
+                 dtype,
+                 name='eq_policy',
+                 **kwargs):
+
+        super().__init__(state_dim=state_dim,
+                         action_dim=action_dim,
+                         name=name,
+                         dtype=dtype,
+                         **kwargs)
+
+        # Number of radial basis functions
+        self.num_eq_features = num_eq_features
+
+        # Set RBF policy locations
+        eq_locs_init = tf.zeros((num_eq_features, state_dim), dtype=dtype)
+        self.eq_locs = UnconstrainedVariable(eq_locs_init,
+                                             name='eq_locs',
+                                             dtype=dtype)
```

Comment:
These variables will need to be re-written to use `ntfo.BoundedVariable`.

Reply:
Let's leave this for later.